### ISSUE TITLE
[V3 ModLog] create_case until field raises exception

### ISSUE THREAD

**Issue by celestialfault** [Cog-Creators/Red-DiscordBot#1486]
Please be sure to read through other issues as well to make sure what you are suggesting/reporting has not already
been suggested/reported

### Type:

- [ ] Suggestion
- [x] Bug

### Brief description of the problem

Attempting to create a case with an until field raises a TypeError exception

### Expected behavior

The case is created normally

### Actual behavior

```py
Traceback (most recent call last):
  File "/home/odinair/redv3/venv/lib64/python3.6/site-packages/redbot/core/dev_commands.py", line 174, in _eval
    result = await func()
  File "<string>", line 8, in func
  File "/home/odinair/redv3/venv/lib64/python3.6/site-packages/redbot/core/modlog.py", line 485, in create_case
    case_content = await case.message_content(use_embeds)
  File "/home/odinair/redv3/venv/lib64/python3.6/site-packages/redbot/core/modlog.py", line 130, in message_content
    end = datetime.fromtimestamp(self.until)
TypeError: an integer is required (got type datetime.datetime)
```

### Steps to reproduce

1. Create a case type
2. Attempt to create a case with an until field

**Comment by palmtree5** [comment]
#1487 should fix this

**Comment by celestialfault** [comment]
Yep, that fixes the issue, thanks :+1:

**Comment by tekulvw** [comment]
Fixed with #1487
